basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/connections/BridgesTestCase.java

```diff
@@ -164,6 +170,25 @@ public void updateBridgeReconnectAttempts() throws Exception {
     }
 
     @Test
+    public void setCredentialReferenceToClearText() throws Exception {
+        page.switchToCredentialReference();
+        elytronChecker.setClearTextCredentialReferenceAndVerify(BRIDGE_ADDRESS, page.getConfigFragment());
+    }
+
+    @Test
+    public void setCredentialReferenceToCredentialStore() throws Exception {
+        page.switchToCredentialReference();
+        elytronChecker.setCredentialStoreCredentialReferenceAndVerify(BRIDGE_ADDRESS, page.getConfigFragment());
+    }
+
+    @Ignore("Ignored until validation to credential-reference is added.")
```

Comment:
Please add to `@Ignore` description the key of HAL enhancement jira to provide client validation.